docs/content/direct/deploy-on-k3d.md

```diff
@@ -0,0 +1,142 @@
+# Deploy KubeStellar on K3D
+
+This document shows how to deploy kubestellar v0.21.0 and a managed cluster on K3D
+
+## Prereqs
+
+In addition to [pre-reqs](pre-reqs.md), install k3d v5.6.0 (only k3d version tested so far)
+
+## Common Setup for standard examples
+
+1. You may want to `set -e` in your shell so that any failures in the setup or usage scenarios are not lost.
+
+1. If you previously installed KS on K3D:
```

Comment:
I don't think we need this section.  If you previously installed KS wit k3d , you don't use this guide :-)